app/src/main/res/layout-v17/clock_fragment.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent">
+
+    <TextClock
+            android:id="@+id/text_clock_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="33sp"
+            android:textColor="#29b6f6"
+            android:format24Hour="HH:mm:ss"
+            android:format12Hour="h:mm:ss"
+            android:timeZone="GMT+0300"
+            android:layout_centerInParent="true"
+            android:layout_alignParentTop="true"
+    />
+
+    <TextClock
+            android:id="@+id/text_date_id"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:textSize="16sp"
+            android:textColor="#786543"
+            android:format24Hour="“EEE, d MMM”"
+            android:format12Hour="“EEE, d MMM”"
+            android:timeZone="GMT+0300"
```

Comment:
Why your timezone is here? You don't need to hardcode it. It need to take time from your system!